theorems/cohomology/CupProduct/OnEM/Commutativity.agda

```diff
@@ -3,455 +3,511 @@
 open import HoTT
 open import homotopy.EilenbergMacLane
 
-module cohomology.CupProduct.OnEM.Commutativity {i} (R : CRing i) where
+module cohomology.CupProduct.OnEM.Commutativity where
```

Comment:
BTW, one thing I was thinking about is to have only lowercased directory names, but don't worry about them for now...